pkg/processor/native/encoder.go

```diff
@@ -0,0 +1,90 @@
+package native
+
+import (
+	"bytes"
+	"errors"
+	"image"
+	"image/jpeg"
+	"image/png"
+)
+
+var DefaultEncoderOptions = &EncoderOptions{
```

Comment:
Would it make sense to rename this to `CompressionOptions`? Just a thought!

Reply:
Yeah that would make sense, will refactor it. Thanks!